EnvironmentSetup.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Setup script which install the Space Concordia Robotics Software team's development environment. 
+
+# Setup venv
+virtualenv -p `which python3.6` venv
```

Comment:
prior to this part there needs to be a few prerequisistes such as

`sudo apt install -y python-pip virtualenv git`

double check where the `-y` goes though

Reply:
And then 
```
sudo add-apt-repository ppa:deadsnakes/ppa
sudo apt update
sudo apt install python3.6
```

Reply:
Additionally it should be checked if `pip` for python3 is installed and if not to install it for the user, maybe like [here](https://pip.readthedocs.io/en/stable/installing/) with a curl and then just running `python3.6 get-pip.py`.

Reply:
Nevermind, I tested it and you absolutely need to do: `sudo apt install python3.6-venv` before using `python3.6 -m venv venv`. See [here](https://github.com/pypa/virtualenv/issues/1059#issuecomment-313867040) for reference. I still think we should change it to `python3.6 -m venv venv` though.

Reply:
@MartensCedric I've only ever seen `-y` at the end of the command but I don't know if that's the only way

Reply:
> Nevermind, I tested it and you absolutely need to do: `sudo apt install python3.6-venv` before using `python3.6 -m venv venv`. See [here](https://github.com/pypa/virtualenv/issues/1059#issuecomment-313867040) for reference. I still think we should change it to `python3.6 -m venv venv` though.

I'm not sure I understand what you want to be changed?  Do me to do an install for python3.6-venv and then `python3.6 -m venv venv` instead of `virtualenv -p 'which python3.6' venv`?

Reply:
Sorry I rambled on a bit there but it's all there:
1. I am suggesting replacing `virtualenv -p which python3.6 venv` with `python3.6 -m venv venv` since if we can possibly avoid an extra debian package install and just use python to create the venv.
2. Incase the python install doesn't includ pip as well, make sure to install pip (specifically for 3.6) if it is not already installed.